src/storybook/config.js

```diff
@@ -101,6 +101,9 @@ function loadStories() {
   require("./stories/pages/PageDetailsPage");
   require("./stories/pages/PageListPage");
 
+  // Page types
+  require("./stories/pageTypes/PageTypeListPage");
```

Comment:
I know it's a mess, but please add story as `.stories.tsx` file in the component's directory, it's a legacy file that I intend to migrate someday